pyproject.toml

```diff
@@ -0,0 +1,78 @@
+[build-system]
+requires = ["setuptools >= 64", "setuptools-scm>=8.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "bluepyopt"
+authors = [
+    {name = "Blue Brain Project, EPFL", email = "werner.vangeit@epfl.ch"},
```

Comment:
I am not sure that they can reach him through this email address anymore

Reply:
I think it is still valid because he is an external collaborator